packages/components/src/popover/index.tsx

```diff
@@ -481,18 +482,19 @@ const UnforwardedPopover = (
 /**
  * `PopoverSlot` is a component that renders a slot that is used by Popover for rendering content.
  */
-export const PopoverSlot = forwardRef(
-	( { name = SLOT_NAME }: { name?: string }, ref: ForwardedRef< any > ) => {
-		return (
-			<Slot
-				bubblesVirtually
-				name={ name }
-				className="popover-slot"
-				ref={ ref }
-			/>
-		);
-	}
-);
+export const PopoverSlot = forwardRef<
+	HTMLDivElement,
+	WordPressComponentProps< PopoverSlotProps, 'div', false >
```

Comment:
No need to use `WordPressComponentProps` here since we're only parsing and using `name`.

```suggestion
	PopoverSlotProps
```